functions/src/SQR.functions.js

```diff
@@ -0,0 +1,234 @@
+
+const functions = require('firebase-functions');
+const admin = require("firebase-admin");
+
+const bucket = admin.storage().bucket();
+const db = admin.database();
+const helpers = require('./../helpers');
+
+
+
+const SQR_Helper = require('./SQR_Helper');
```

Comment:
Do we really need a separate helper for each module? Why not moving all the SQR-related code into SQR file? Just asking what is behind your decision.

Reply:
No, you're right ... in the beginning I thought it would be better to keep the SQR functions clean by having the calling separate functions with names describing what they do .. like this for example
```javascript
exports.syncStorageToDB = functions.https.onRequest((req, res) => {
    SQR_Helper.handleCurrentlyUploadedFiles(bucket, db, helpers.storeFileNameToDB);
    SQR_Helper.removeNonExistingMp3DBEntries(bucket, db, helpers.removeFileNameFromDB);
    return res.send(`Started Execution, the process is now Running in the background`);
});
```

but I think you're right .. it's useless, I can replace this with comments before every function

Reply:
Yes. And consider using classes if possible and reasonable.

Reply:
in Node.js 8 ( the latest in Firebase functions ) I can't use the **`export`** or **`import`** keywords... So I don't think it would make sense to use classes now.